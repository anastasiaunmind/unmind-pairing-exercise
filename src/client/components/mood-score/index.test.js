import React from "react";
import { render } from "@testing-library/react";
import MoodScore from ".";

test("renders MoodScore", () => {
    const { container } = render(<MoodScore score={4} onChange={() => {}} />);

    expect(container).toMatchSnapshot();
});

// todo unit tests