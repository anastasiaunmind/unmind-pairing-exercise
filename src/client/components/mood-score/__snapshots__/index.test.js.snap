// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders MoodScore 1`] = `
<div>
  <div
    class="sc-bdnxRM fJSvQQ"
    data-testid="score"
  >
    <svg
      height="118.48516707043927"
      preserveAspectRatio="xMidYMid meet"
      version="1.1"
      viewBox="210.8018018018018 97.82882882882876 165.98198198198207 222.76844930036418"
      width="161"
      xmlns="http://www.w3.org/2000/svg"
      xmlns:xlink="http://www.w3.org/1999/xlink"
    >
      <defs>
        <path
          d="M265.59 155.59C265.59 166.62 256.62 175.59 245.59 175.59C234.55 175.59 225.59 166.62 225.59 155.59C225.59 144.55 234.55 135.59 245.59 135.59C256.62 135.59 265.59 144.55 265.59 155.59Z"
          id="b4Cbfd7GO"
        />
        <path
          d="M360 157.39C360 168.43 351.04 177.39 340 177.39C328.96 177.39 320 168.43 320 157.39C320 146.35 328.96 137.39 340 137.39C351.04 137.39 360 146.35 360 157.39Z"
          id="b4acjEWOqz"
        />
        <path
          d="M311.92 240.23C311.92 250.26 302.95 258.4 291.92 258.4C280.88 258.4 271.92 250.26 271.92 240.23C271.92 230.21 280.88 222.07 291.92 222.07C302.95 222.07 311.92 230.21 311.92 240.23Z"
          id="cc7T99TZt"
        />
        <path
          d="M272.43 99.73C276.26 99.73 279.37 102.84 279.37 106.67C279.37 108.05 279.37 105.28 279.37 106.67C279.37 110.5 276.26 113.6 272.43 113.6C260.31 113.6 230.86 113.6 218.74 113.6C214.91 113.6 211.8 110.5 211.8 106.67C211.8 105.28 211.8 108.05 211.8 106.67C211.8 102.84 214.91 99.73 218.74 99.73C230.86 99.73 260.31 99.73 272.43 99.73Z"
          id="b2SIuDtO2z"
        />
        <path
          d="M366.85 98.83C370.68 98.83 373.78 101.93 373.78 105.77C373.78 107.15 373.78 104.38 373.78 105.77C373.78 109.6 370.68 112.7 366.85 112.7C354.72 112.7 325.28 112.7 313.15 112.7C309.32 112.7 306.22 109.6 306.22 105.77C306.22 104.38 306.22 107.15 306.22 105.77C306.22 101.93 309.32 98.83 313.15 98.83C325.28 98.83 354.72 98.83 366.85 98.83Z"
          id="a5dELl33Zv"
        />
        <path
          d="M368.1 303.72C370.02 303.72 371.57 305.28 371.57 307.19C371.57 309.27 371.57 312.05 371.57 314.13C371.57 316.04 370.02 317.6 368.1 317.6C338.01 317.6 251.2 317.6 221.1 317.6C219.19 317.6 217.63 316.04 217.63 314.13C217.63 312.05 217.63 309.27 217.63 307.19C217.63 305.28 219.19 303.72 221.1 303.72C251.2 303.72 338.01 303.72 368.1 303.72Z"
          id="a3CYeoj5"
        />
      </defs>
      <g>
        <g>
          <use
            fill="#60d200"
            fill-opacity="1"
            opacity="1"
            xlink:href="#b4Cbfd7GO"
          />
        </g>
        <g>
          <use
            fill="#60d200"
            fill-opacity="1"
            opacity="1"
            xlink:href="#b4acjEWOqz"
          />
        </g>
        <g>
          <use
            fill="#bababa"
            fill-opacity="1"
            opacity="1"
            xlink:href="#cc7T99TZt"
          />
        </g>
        <g>
          <use
            fill="#bababa"
            fill-opacity="1"
            opacity="1"
            xlink:href="#b2SIuDtO2z"
          />
        </g>
        <g>
          <use
            fill="#bababa"
            fill-opacity="1"
            opacity="1"
            xlink:href="#a5dELl33Zv"
          />
        </g>
        <g>
          <use
            fill="#60d200"
            fill-opacity="1"
            opacity="1"
            xlink:href="#a3CYeoj5"
          />
        </g>
      </g>
    </svg>
  </div>
</div>
`;
